src/components/SearchModal/ImportToken.tsx

```diff
@@ -47,22 +50,25 @@ export function ImportToken(props: ImportProps) {
       <PaddedColumn gap="14px" style={{ width: '100%', flex: '1 1' }}>
         <RowBetween>
           {onBack ? <ArrowLeft style={{ cursor: 'pointer' }} onClick={onBack} /> : <div />}
-          <ThemedText.MediumHeader>
-            <Plural value={tokens.length} one="Import token" other="Import tokens" />
-          </ThemedText.MediumHeader>
+          <ThemedText.MediumHeader>Import token</ThemedText.MediumHeader>
           {onDismiss ? <CloseIcon onClick={onDismiss} /> : <div />}
         </RowBetween>
       </PaddedColumn>
       <SectionBreak />
       <AutoColumn gap="md" style={{ marginBottom: '32px', padding: '1rem' }}>
         <AutoColumn justify="center" style={{ textAlign: 'center', gap: '16px', padding: '1rem' }}>
           <AlertCircle size={48} stroke={theme.text2} strokeWidth={1} />
-          <ThemedText.Body fontWeight={400} fontSize={16}>
-            <Trans>
-              This token doesn&apos;t appear on the active token list(s). Make sure this is the token that you want to
-              trade.
-            </Trans>
-          </ThemedText.Body>
+          {chainId && (
+            <ThemedText.Body fontWeight={400} fontSize={16}>
+              <Trans>
+                This token is not on your active token lists. Please check the{' '}
+                <ExternalLink href={getExplorerLink(chainId, tokens[0].address, ExplorerDataType.ADDRESS)}>
```

Comment:
I'm cheating again here. Open to a less hacky solution to get the token's address. This works for the common (only afaik?) use case of importing a single token.